src/Components/Todolist.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface TodoListPropsType {
+  todosToShow: Todo[],
+  todoDelete: (todoId: number) => void,
+  deletedId: number,
+  tempTodo: Todo,
+  added: boolean,
+}
+
+export const TodoList: React.FC<TodoListPropsType> = ({
+  todosToShow,
+  todoDelete,
+  deletedId,
+  tempTodo,
+  added,
+}) => {
+  if (added) {
+    todosToShow.push(tempTodo);
+  }
```

Comment:
it's not the best solution to modify the property. please create the new one
```suggestion
  const todosShown = added ? [...todosToShow, tempTodo] : todosToShow;
```

Reply:
Hi Mam. It's changing the todo add animation, So I didn't